mobile-widgets/qmlinterface.h

```diff
@@ -81,9 +81,7 @@ class QMLInterface : public QObject {
 	Q_PROPERTY(QStringList diveSummaryText READ diveSummaryText NOTIFY diveSummaryTextChanged);
 
 public:
-	static QMLInterface *instance();
-
-	// function to do the needed setup and do connect of signal/signal
+	// function to do the needed setup
```

Comment:
This is not wrong, but merely a matter of taste. I actually use the public instance() quite a lot when I test and prepare new commits.